### ISSUE TITLE
Pre-built development library doesn't update paths for CMake files

### ISSUE THREAD

**Issue by ccawley2011** [libsdl-org/SDL#4481]
The pre-built development library archive for MinGW comes with a Makefile to update the paths for the .la, .pc and sdl2-config files when installing the library, however it doesn't update the CMake files, which prevents them from working correctly.

**Comment by slouken** [comment]
Can you attach a patch to fix this for you?

Thanks!

**Comment by ccawley2011** [comment]
This patch works for me:

```
--- Makefile.orig       2018-01-29 06:12:13 +0000
+++ Makefile    2021-06-30 19:44:17 +0100
@@ -23,6 +23,8 @@
            chmod 755 $(prefix)/bin/sdl2-config; \
            sed "s|^libdir=.*|libdir=\'$(prefix)/lib\'|" <$(arch)/lib/libSDL2.la >$(prefix)/lib/libSDL2.la; \
            sed "s|^libdir=.*|libdir=\'$(prefix)/lib\'|" <$(arch)/lib/libSDL2main.la >$(prefix)/lib/libSDL2main.la; \
+           sed "s|^libdir=.*|libdir=\'$(prefix)/lib\'|" <$(arch)/lib/libSDL2_test.la >$(prefix)/lib/libSDL2_test.la; \
+           sed -e "s|^set(prefix "".*"")|set(prefix \"$(prefix)\")|" -e "s|^set(SDL2_PREFIX "".*"")|set(SDL2_PREFIX \"$(prefix)\")|" -e "s|^set(SDL2_EXEC_PREFIX "".*"")|set(SDL2_EXEC_PREFIX \"$(prefix)\")|" <$(arch)/lib/cmake/SDL2/sdl2-config.cmake >$(prefix)/lib/cmake/SDL2/sdl2-config.cmake; \
            sed "s|^prefix=.*|prefix=$(prefix)|" <$(arch)/lib/pkgconfig/sdl2.pc >$(prefix)/lib/pkgconfig/sdl2.pc; \
        else \
            echo "*** ERROR: $(arch) or $(prefix) does not exist!"; \
```

**Comment by slouken** [comment]
This is fixed for the 2.0.16 release, thanks!

**Comment by ccawley2011** [comment]
This issue seems to have returned with the SDL 2.0.20 release archives. Here's a newer version of the above patch:
```
--- Makefile.orig	2018-01-29 06:12:13 +0000
+++ Makefile	2022-04-07 14:24:57 +0100
@@ -23,7 +23,9 @@
 	    chmod 755 $(prefix)/bin/sdl2-config; \
 	    sed "s|^libdir=.*|libdir=\'$(prefix)/lib\'|" <$(arch)/lib/libSDL2.la >$(prefix)/lib/libSDL2.la; \
 	    sed "s|^libdir=.*|libdir=\'$(prefix)/lib\'|" <$(arch)/lib/libSDL2main.la >$(prefix)/lib/libSDL2main.la; \
+	    sed "s|^libdir=.*|libdir=\'$(prefix)/lib\'|" <$(arch)/lib/libSDL2_test.la >$(prefix)/lib/libSDL2_test.la; \
 	    sed "s|^prefix=.*|prefix=$(prefix)|" <$(arch)/lib/pkgconfig/sdl2.pc >$(prefix)/lib/pkgconfig/sdl2.pc; \
+	    sed "s|^set(prefix "".*"")|set(prefix \"$(prefix)\")|" <$(arch)/lib/cmake/SDL2/sdl2-config.cmake >$(prefix)/lib/cmake/SDL2/sdl2-config.cmake; \
 	else \
 	    echo "*** ERROR: $(arch) or $(prefix) does not exist!"; \
 	    exit 1; \
```
